wremnants/theory_tools.py

```diff
@@ -93,10 +93,13 @@ def qcdScaleNames():
     shifts = ["muRmuFDown", "muRDown", "", "muFDown", "", "muFUp", "muRUp", "", "muRmuFUp"]
     return ["_".join(["QCDscale", s]) if s != "" else s for s in shifts]
 
-def massWeightNames(matches=None):
+def massWeightNames(matches=None, wlike=False):
     central=10
     nweights=21
     names = [f"massShift{int(abs(central-i)*10)}MeV{'Down' if i < central else 'Up'}" for i in range(nweights)]
+    if wlike:
+        # These are the Z width variations
+        names.extend(["", ""])
```

Comment:
Why are these added as empty strings?

Reply:
See the comment right below this

Reply:
The line right below this tries to explain it. Ultimately each variation gets projected, those with empty names don't get stored in the output root file.